WebRoot/templates/abook/Contacts.template

```diff
@@ -1025,6 +1025,13 @@
 											<td id='${id}_listSelect'></td>
 										</tr>
 									</table>
+								<$ } else { $>
+									<table role="presentation" class='ZPropertySheet' cellspacing='6' style="visibility: hidden;">
+										<tr>
+											<td class='Label nobreak' id='${id}_listSelectLbl_invisible'><$= ZmMsg.showNames $></td>
+											<td id='${id}_listSelect_invisible'><$= ZmMsg.searchPersonalSharedContacts $></td>
+										</tr>
+									</table>
```

Comment:
* `style="visibility: hidden;"` is set for the table.
* comparing with code at lines 1022-1027, different classname is set on each so that the change will not affects any js code which getting an element of `_listSelectLbl` or `_listSelect`
    * `_listSelectLbl` -> `_listSelectLbl_invisible`
    * `_listSelect` -> `_listSelect_invisible`